Configure error: JsonCpp_INCLUDE_DIR does NOT match value used to configure Slicer

Many extensions have this new configuration error due to calling this in a module's CMakeLists.txt:

```
find_package(Slicer REQUIRED)
include(${Slicer_USE_FILE})
```

Resulting error message:

```
-- Configuring Loadable module: WorkflowSegmentation [qSlicerWorkflowSegmentationModuleExport.h]
CMake Error at D:/D/P/Slicer-0-build/Slicer-build/SlicerConfig.cmake:681 (message):
  Variable JsonCpp_INCLUDE_DIR defined prior calling 'find_package(Slicer)'
  does NOT match value used to configure Slicer.  It probably means that a
  different JsonCpp_INCLUDE_DIR has been used to configure this project and
  Slicer.

  JsonCpp_INCLUDE_DIR=

  Slicer_JsonCpp_INCLUDE_DIR=D:/D/P/Slicer-0-build/JsonCpp/include
Call Stack (most recent call first):
  D:/D/P/Slicer-0-build/Slicer-build/SlicerConfig.cmake:879 (slicer_config_set_ep)
  SkillAssessment/CMakeLists.txt:10 (find_package)


-- Configuring incomplete, errors occurred!
See also "D:/D/P/S-0-E-b/PerkTutor-build/CMakeFiles/CMakeOutput.log".
See also "D:/D/P/S-0-E-b/PerkTutor-build/CMakeFiles/CMakeError.log".
```

(http://slicer.cdash.org/viewConfigure.php?buildid=1979043)

The error is fixed by removing find_package+include (see [fix in SlicerHeart](https://github.com/SlicerHeart/SlicerHeart/commit/7b81faac7cd723e34cb020914e0ed796be30c1bc)).

It would be better to fix this in Slicer so that there is no need to update extensions and templates.

--- a/SuperBuild/External_SlicerExecutionModel.cmake
+++ b/SuperBuild/External_SlicerExecutionModel.cmake
@@ -73,7 +73,7 @@ if(NOT DEFINED SlicerExecutionModel_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "1788b378ed2e4928cded2bc9ecdc2b37c7f2af5f"
+    "f19d6e88a94ba8f31ddafcff4adf185fe90d7e72"
     QUIET
     )
 

